### DISCUSSION TITLE
[4.x] Jersey support

### DISCUSSION THREAD

**Discussion by mcopes73** [helidon-io/helidon#7693] [Q&A] [answered]
Is [Jersey support](https://helidon.io/docs/v2/apidocs/io.helidon.webserver.jersey/io/helidon/webserver/jersey/JerseySupport.html) in the roadmap for SE 4.x? It would be useful to have a utility to translate existing Jersey Features. Same for existing services annotated with `jakarta.ws.rs` annotations. Is there an easy way to adapt them to Helidon 4.x services?

**Comment by romain-grecourt** [comment, marked as answer]
In 4.x, `JerseySupport` has been moved as an non public implementation included in Helidon MP. There is no plan to support Jersey in Helidon SE going forward.
